linux/README.md

```diff
@@ -3,17 +3,23 @@ This is a linux docker image that uses python to download the SAClientUtil from
 
 ### Variables
 
-The pipe has 8 variables.
+The pipe has 13 variables.
 
 | Variable |  Required | Description |
 |---|---|---|
 | API_KEY_ID | Required | The HCL AppScan on Cloud API Key ID |
 | API_KEY_SECRET | Required | The HCL AppScan on Cloud API Key Secret |
 | APP_ID | Required | The application Id of the app in AppScan on Cloud |
 | TARGET_DIR | Required | The directory to be scanned. Place scan targets here. |
+| CONFIG_FILE_PATH | Optional | Relative path from the repo root to an appscan config xml file. |
+| SECRET_SCANNING | Optional | True or False (default). Enables the secret scanning feature of ASoC SAST. |
 | BUILD_NUM | Optional | The Bitbucket build number. Used to make filenames and comments relevant. |
+| REPO | Optional | The Repository name. Only really used to make filenames and comments relevant. |
```

Comment:
Can we get this value in another way without the user specifying it? What's the name if it's not specified?

Reply:
If not specified we get the repo name from the BITBUCKET_REPO env variables set by the pipeline.  I'm not 100% sure how setting this to another repo would affect the scan, it was added as part of a past change.  I'm keeping the readme up to date.